internal/config/config_save.go

```diff
@@ -0,0 +1,23 @@
+package config
+
+import (
+	"os"
+
+	"github.com/BurntSushi/toml"
+)
+
+const configPerm os.FileMode = 0o640
+
+func (c *Config) Save() error {
+	path, err := c.configFilePath()
+	if err != nil {
+		return err
+	}
+
+	w, err := os.OpenFile(path, os.O_CREATE|os.O_WRONLY|os.O_TRUNC, configPerm)
```

Comment:
shouldn't this defer w.Close()?

Reply:
Good point!

Reply:
I at least checked If it was necessary, it should be fine, but can cause issues with buffer not getting flushed. And that would be a bit unfortunate when they think they saved it :)

Reply:
I added it 😄